test/dummy_folder/undeletable

```diff
@@ -0,0 +1 @@
+I cannot be deleted. Haha .. Haha
```

Comment:
Why can't this be deleted?

Reply:
I created this file using root permission in my system. This file can only be deleted using root permission. I used --> sudo chattr +i filename, to protect it. This file was created to test the function fs.move function that deletes the source file after it copies the content to destination and returns true, but in this case it returns false because the file cannot be deleted.

Reply:
git does not record that fact: someone cloning the repository will not have that attribute.
You'll need to create a file with that attribute as part of the test.